dispatch/static/manager/src/js/constants/ActionTypes.js

```diff
@@ -11,7 +11,7 @@ export const TOPICS = resourceActionTypes('TOPICS')
 export const TAGS = resourceActionTypes('TAGS')
 export const TEMPLATES = resourceActionTypes('TEMPLATES')
 export const GALLERIES = resourceActionTypes('GALLERIES')
-export const EVENTS = resourceActionTypes('EVENTS', ['COUNT_PENDING'])
+export const EVENTS = resourceActionTypes('EVENTS', ['COUNT_PENDING', 'PUBLISH', 'UNPUBLISH'])
```

Comment:
`PUBLISH` and `UNPUBLISH` action types aren't used, so they aren't necessary here.